rules/eslint/best-practices/walmart.js

```diff
@@ -26,12 +26,16 @@ module.exports = {
     "no-alert": 2,
     // disallow use of arguments.caller or arguments.callee
     "no-caller": 2,
+    // disallow lexical declarations in case clauses
+    "no-case-declarations": 0,
     // disallow division operators explicitly at beginning of regular expression
     "no-div-regex": 0,
     // disallow else after a return in an if
     "no-else-return": 0,
     // disallow use of labels for anything other then loops and switches
     "no-empty-label": 2,
+    // disallow use of empty destructuring patterns
+    "no-empty-pattern": 0,
```

Comment:
This seems like quite a gotcha and I'd like to turn it on: http://eslint.org/docs/rules/no-empty-pattern